engineapi/engineapi/settings.py

```diff
@@ -198,3 +198,11 @@
 MOONSTREAM_ADMIN_ID = os.environ.get("MOONSTREAM_ADMIN_ID", "")
 if MOONSTREAM_ADMIN_ID == "":
     raise ValueError("MOONSTREAM_ADMIN_ID environment variable must be set")
+
+MOONSTREAM_LEADERBOARD_GENERATOR_JOURNAL_ID = os.environ.get(
```

Comment:
Can we call this MOONSTREAM_LEADERBOARD_CONFIGURATION_JOURNAL_ID?

Reply:
+1

Reply:
changed